2_preference_alignment/dpo.md

```diff
@@ -16,38 +16,57 @@ Next comes preference learning, where the model is trained on pairs of outputs -
 
 The core innovation of DPO lies in its direct optimization approach. Rather than training a separate reward model, DPO uses a binary cross-entropy loss to directly update the model weights based on preference data. This streamlined process makes training more stable and efficient while achieving comparable or better results than traditional RLHF.
 
+## DPO datasets
+
+Datasets for DPO are typically created by annotating pairs of responses as preferred or non-preferred. This can be done manually or using automated filtering techniques. Below is an example structure of a DPO dataset:
+
+| Input | Prompt | Chosen | Rejected |
```

Comment:
AFAIK input is not really required and may lead to confusion (?), I think following the standard prompt, chosen, rejected makes the most sense at least for single-turn, whilst for multi-turn you could use messages, chosen, and rejected instead too.